src/main/protobuf/build/buildfarm/v1test/buildfarm.proto

```diff
@@ -1062,6 +1065,9 @@ message WorkerConfig {
 
   // user principal name for executions
   string exec_owner = 29;
+
+  // a limit on time (in seconds) for input fetch stage to fetch inputs
+  int32 input_fetch_deadline = 30;
```

Comment:
we probably want to consolidate everything to `google.protobuf.Duration` in the future